src/components/pages/devices/flyouts/deviceDetails/deviceDetails.js

```diff
@@ -342,18 +344,19 @@ export class DeviceDetails extends Component {
                         <Cell className="col-7">{device.connected ? t('devices.flyouts.details.connected') : t('devices.flyouts.details.notConnected')}</Cell>
                       </Row>
                       {
-                        device.connected && [
+                        device.connected &&
+                        <ComponentArray>
                           <Row key="diag-row-time">
                             <Cell className="col-3">{t('devices.flyouts.details.diagnostics.lastMessage')}</Cell>
-                            <Cell className="col-7">{ lastMessageTime ? moment(lastMessageTime).format(DEFAULT_TIME_FORMAT) : '---' }</Cell>
+                            <Cell className="col-7">{lastMessageTime ? moment(lastMessageTime).format(DEFAULT_TIME_FORMAT) : '---'}</Cell>
                           </Row>,
                           <Row key="diag-row-msg">
```

Comment:
`key` not needed

Reply:
also comma in line above this is wrong too